src/fibaroAccessory.ts

```diff
@@ -129,6 +129,12 @@ export class FibaroAccessory {
             this.platform.Characteristic.SecuritySystemTargetState];
           subtype = '0--';
           break;
+        case 'airQualitySensorPm25':
+          service = this.platform.Service.AirQualitySensor;
+          this.mainCharacteristics = [this.platform.Characteristic.AirQuality,
+            this.platform.Characteristic.PM2_5Density];
+          subtype = device.id + '--PM2_5';
```

Comment:
I don't think subtype is needed for the device? This is probably only for virtual things like scene, climate zone, security, etc.

Reply:
Ok, now i know that this is for future, for other types of air quality sensors.